src/test/resources/org/jenkinsci/plugins/workflow/flow/FlowExecutionListTest/stepExecutionIteratorDoesNotLeakBuildsWhenOneIsStuck/jobs/test0/builds/1/workflow/7.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<Tag plugin="workflow-support@907.v6713a_ed8a_573">
+  <node class="cps.n.StepEndNode" plugin="workflow-cps@3894.3896.vca_2c931e7935">
+    <parentIds>
+      <string>6</string>
+    </parentIds>
+    <id>7</id>
+    <startId>8</startId> <!-- Manually edited to simulate corruption observed in the real world. -->
```

Comment:
In the real build that I saw, this pointed to the `lock` step from `lockable-resources`, and there were no other nodes past that `StepStartNode`. I am still quite curious as to how we got a `startId` that was newer than `id` though whenever things got corrupted for the first time.